react.js

```diff
@@ -1,26 +1,23 @@
-/* eslint-disable global-require */
-
 const React = require('react')
 const reactTestingLibrary = require('@testing-library/react')
-const reactHooksTestingLibrary = require('@testing-library/react-hooks')
+const hooks = require('@testing-library/react-hooks')
 const { MockedProvider } = require('@apollo/react-testing')
 const { IntlProvider } = require('react-intl')
-const { path, find, isNil, reject } = require('ramda')
 const { InMemoryCache } = require('apollo-cache-inmemory')
 
 const paths = require('./modules/paths')
 
 const pkg = require(paths.resolveAppPath('package.json'))
 
 const getLocale = (optionsLocale) => {
-  const pkgLocale = path(['vtexTestTools', 'defaultLocale'], pkg)
+  const pkgLocale = pkg.vtexTestTools.defaultLocale
```

Comment:
Any null reference risk?

Reply:
The previous code wasn't checking for any fallback value 🤷 And we're testing for its truthiness two lines below!

Reply:
But I see, `vtexTestTools` could be undefined, will add a check for that!

Reply:
What about a thrown error? Just to be sure 😬